tools/release.rb

```diff
@@ -101,8 +101,12 @@ def self.generate_release_version(release_type)
 
   private
 
-  def remove_old_releases
-    FileUtils.rm Dir.glob(File.join(Machinery::ROOT, "package/*"))
+  def remove_old_releases(skip_cleanup)
```

Comment:
`package/*.gem` and `/var/tmp/.../*.rpm` are removed although the `skip_cleanup` option is set. 
If that's on purpose, the option name is misleading.